ChatServices/MattermostChatServiceConnection.cs

```diff
@@ -478,6 +511,29 @@ public async Task<string> GetChannelIdFromChannelNameAsync(string channelName)
             }
         }
 
+        public async Task<int> GetMemberCountFromChannelIdAsync(string channelId)
+        {
+            // note: this currently only counts users from the first page of results, since this is sufficient for our current use cases
```

Comment:
It's _paginated_?? Thanks Mattermost 🤦‍♂️

Reply:
The docs weren't very clear, I am not sure if the API meant that it is always paginated or that you can choose to fetch the data paginated if you wanted. I just left this comment as a "worst case" scenario.